cmd/status.go

```diff
@@ -123,6 +129,10 @@ own. You can specify the user as "all" to see all user's sets.
 			die("--remote can only be used with --details and --name")
 		}
 
+		if statusOrder != "" && statusOrder != "alphabetic" && statusOrder != "recent" {
```

Comment:
Don't need the blank check if you set a default value, and "alphabetic" and "recent" should be constants used throughout rest of code.